src/Middleware/RateLimiting/src/RateLimitingOptions.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading.RateLimiting;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.RateLimiting;
+
+/// <summary>
+/// Specifies options for the <see cref="RateLimitingMiddleware"/>.
+/// </summary>
+public class RateLimitingOptions
+{
+    // TODO - Provide a default?
+    private PartitionedRateLimiter<HttpContext> _limiter = new NoLimiter<HttpContext>();
```

Comment:
This seems problematic. You'd want to either pick good defaults or require a user to configure it (possibly by throwing a runtime exception if this is left unconfigured)

Reply:
We're planning to later add a default implementation that will look for endpoint metadata to determine whether or not a rate limit should be applied to the request. It will be a lot like auth or cors middleware. Similarly to auth, we'll probably not limit requests to endpoints without any metadata unless overridden by global policy.